### PR TITLE
[10.0] [ADD] stock_analytic_change_product_qty

### PR DESCRIPTION
**Pull request by nikul-serpentcs**
Stock Location Analytic Change Product Qty
==================================

 When updating the product qty referring to an analytic account the location must contain the same analytic account

### CODE REVIEW

**Comment by nikul-serpentcs** [comment]
@aheficent Could you please review.

File: stock_location_analytic_change_product_qty/README.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Stock Analytic
```
Comment (AaronHForgeFlow) [inline stock_location_analytic_change_product_qty/README.rst]: Please change this by the correct title

File: stock_location_analytic_change_product_qty/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
```
Comment (AaronHForgeFlow) [inline stock_location_analytic_change_product_qty/__init__.py]: Reserved for authors of people who introduce important new features

**Review by AaronHForgeFlow — COMMENTED** [review]
Functional ok, only some style issues. Besides, I'm not sure if the commit history is respected here

File: stock_location_analytic_change_product_qty/README.rst [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+==============
+
+ When updating the product qty referring to an analytic account the location
+ must contain the same analytic account
+
+
+Usage
+=====
+
+Update the product qty as always
```
Comment (AaronHForgeFlow) [inline stock_location_analytic_change_product_qty/README.rst]: Below this line add
* Aaron Henriquez <ahenriquez@eficent.com> and yourself as contributors

**Comment by nikul-serpentcs** [comment]
@aheficent Could you please look it.

**Comment by nikul-serpentcs** [comment]
@aheficent Could you please look it.

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
In the wizard I cannot select a location out of the company. We should be able to select also "customer" locations that are associated to an analytic account. Why? Because I don't those products to count as an asset in my warehouse, but the products are already there, and they are assigned to a project. The stock_analytic_location module allows me to do that

**Comment by nikul-serpentcs** [comment]
@aheficent Could you please look it.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1:  Thanks for attending the review. Can you please squash you commit in just one?

**Comment by nikul-serpentcs** [comment]
@aheficent squashed commit, Could you please look it.

File: stock_location_analytic_change_product_qty/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Update Product Qty Analytic',
+    'version': '10.0.1.0.0',
+    'category': 'Analytic',
+    'summary': 'Update product Qty when using Analytic locations',
+    'author': 'Eficent',
+    'website': 'http://www.eficent.com',
```
Comment (AaronHForgeFlow) [inline stock_location_analytic_change_product_qty/__manifest__.py:11]: Can you add the license key here? 

Sorry i did not see that before

**Comment by nikul-serpentcs** [comment]
@aheficent Could you please look it.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1:

**Pull request merged**